cmd/awat/translation.go

```diff
@@ -145,20 +145,19 @@ var startTranslationCmd = &cobra.Command{
 		}
 
 		var err error
+		var uploadID *string
```

Comment:
We can use `string` here, and pass/set pointer whenever needed.

Reply:
It could be `nil` as well (and that's the default if no `--upload` flag is provided).